From b05d0ce99fe15872b227c75917cfe8e754415b9f Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Wed, 4 Oct 2023 17:44:46 +0200
Subject: [PATCH 040/184] 
 linux-mainline-and-mali-allwinner-h6-kernel/misc.616/patches/add-own-extra-box-dtb-files.patch

---
 arch/arm64/boot/dts/allwinner/Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/Makefile b/arch/arm64/boot/dts/allwinner/Makefile
index 76fcb7a15..694cec965 100644
--- a/arch/arm64/boot/dts/allwinner/Makefile
+++ b/arch/arm64/boot/dts/allwinner/Makefile
@@ -44,8 +44,14 @@ dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-tanix-tx6s.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-t95.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h313-x96q.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-tx6s.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-box.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-box-30mv.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-box-60mv.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-lemfo.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-t95max.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-box.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-box-30mv.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-box-60mv.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h313-x96-q-lpddr3.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-tanix-tx6s-axp313.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-vontar-h618.dtb
-- 
2.39.2

